doc/devel/development_setup.rst

```diff
@@ -133,3 +133,9 @@ Optional, but recommended:
   built without Inkscape and optipng, but the build process will raise various
   warnings. If the build process warns that you are missing fonts, make sure
   your LaTeX distribution bundles cm-super or install it separately.
+
+Troubleshooting
+===============
+*  pytest - If the test runner is failing, verify that the pytest you are using
+is installed in your environment using the ``which pytest`` and ``which python``
+commands. If the paths are not the same, install pytest in your environment. 
```

Comment:
I have doubts about troubleshooting / FAQ sections. My impression is that they get cluttered with the next best thing someone comes across and accumulate lots of clutter between relevant information.

In this specific case:

- do we need this at all, if we suggest `python -m pytest`?
- If so, this should move to `testing.rst`. While it's a dependency error, users won't know that. It will happen in the context of running tests, so that's the first place to look at in the docs.

Reply:
what about having a troubleshooting section in the wiki, and linking to it from the docs? Would be easier to maintain, but still provide info.

Reply:
I mostly agree with @timhoffm here: we should just use `python -mpytest` everywhere and perhaps just document "always use `python -mpytest`, not `pytest`", not end up with neverending faq entries.

Reply:
changed to -m pytest and dropped the troubleshooting